pkg/controller/import-controller.go

```diff
@@ -525,6 +525,21 @@ func (r *ImportReconciler) createImporterPod(pvc *corev1.PersistentVolumeClaim)
 		vddkImageName:     vddkImageName,
 		priorityClassName: getPriorityClass(pvc),
 	}
+
+	// Check for old terminating scratch spaces
+	if requiresScratch && podArgs.scratchPvcName != nil {
+		r.log.V(1).Info("Checking for old scratch pvcs")
+		scratchPvc := &corev1.PersistentVolumeClaim{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: pvc.GetNamespace(), Name: *podArgs.scratchPvcName}, scratchPvc)
+		if IgnoreNotFound(err) != nil {
+			return err
+		}
+		if scratchPvc.DeletionTimestamp != nil {
+			// Old scratch space with same name is still being cleaned up, return error.
+			return errors.New("Waiting for scratch space to be deleted.")
```

Comment:
lint says you should remove the "." at the end.

Reply:
Done